boards/arm/gd32l233r_eval/gd32l233r_eval_defconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2022 BrainCo Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_GD32L23X=y
+CONFIG_SOC_GD32L233=y
+CONFIG_BOARD_GD32L233R_EVAL=y
+
+
```

Comment:
Duplicated blank line.
Should be better to remove it.

Reply:
Thanks! remove the duplicated line.